mod_project/helpers.py

```
@@ -18,6 +18,28 @@
+def _write_spliney(layer: 'Layer', spliney_id: str, entry: dict):
+    raw = layer.raw_collection('splineys', create=True)
+    raw[spliney_id] = (
+        layer.spliney_for_native(entry)
+        if layer.is_fuse_native else _copy.deepcopy(entry)
+    )
+    layer.remove_world_removal('splineys', spliney_id)
+    layer.splineys[spliney_id] = _copy.deepcopy(entry)
+    layer.dirty = True
+
+
+def _delete_spliney(layer: 'Layer', spliney_id: str):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_delete_spliney`

Add return type annotation: `None`

(ANN202)